build/StoreSubmission/Preview/PDPs/ru-RU/PDP.xml

```diff
@@ -57,8 +57,8 @@
     Версия __VERSION_NUMBER__
 
 - Мы добавили в пользовательский интерфейс десятки параметров, которые существовали только в JSON-файле, включая новую страницу для настройки макета меню "Новая вкладка".
-- Для повышения надежности мы переупоряхлили управление окнами; создайте все ошибки, обнаруженные с wt.exe псевдонимом
-- Профили теперь показывают значок, если они скрыты или ссылаются на программы, которые были удалены.
+- Для повышения надежности мы перепроектировали управление окнами; пожалуйста сообщайте о любых ошибках с которыми столкнетесь связанных с псевдонимом wt.exe

```

Comment:
This sentence is not consistent.

It will be correct either:
```
- Для повышения надежности, мы перепроектировали управление окнами; пожалуйста, сообщайте о любых ошибках в работе Windows Terminal
```
or:
```
- Для повышения надежности, мы перепроектировали управление окнами; пожалуйста, сообщайте о любых ошибках, связанных с использованием псевдонима wt.exe
```
or:
```
- Мы перепроектировали управление окнами для повышения надежности; пожалуйста, сообщайте о любых ошибках, связанных с использованием псевдонима wt.exe
```

Pay attention to the commas.